AGENTS.md

```
@@ -0,0 +1,134 @@
+npm start           # Dev server at http://localhost:3000
+npm run build       # Production build
+npm test            # Interactive Jest test runner
+npm run test:coverage  # Coverage report (80% threshold enforced)
+npm run lint        # ESLint check
+npm run lint:fix    # Auto-fix linting issues
+```
+
+## Local Development Setup
+
+1. **Start infrastructure dependencies** (PostgreSQL, Redis, Kafka, mock services):
+   ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)